src/modal.service.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { TestBed, inject } from '@angular/core/testing';
+
+import { ModalService } from './modal.service';
+
+describe('ClientService', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [ModalService]
+    });
+  });
+
+  it('should be created', inject([ModalService], (service: ModalService) => {
```

Comment:
Redundant boilerplate. Better test might have to be written later, either unit/integration or E2E.

Reply:
help on this. this was the ng generate boilerplate

Reply:
the service has been removed